weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/TraceTreeSplitPanel.tsx

```diff
@@ -0,0 +1,224 @@
+/**
```

Comment:
hmmm ... how much of this is a copy of `weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SplitPanelRight.tsx`? Is there a way to make this generic? ... ie: `SplitPanel` that has a left/right param?

Reply:
I tried, but there are a ton of specific calculations that made the combo component hard to understand. I can try again though, it worked, was just complicated.

Reply:
Ok. It is possible that generalizing makes it harder to maintain or easier.... , up to you. i would at the very least make a comment saying that this is a sister to the other component. Or even put them in the same file / subdir to emphasize their similarity. Furthermore, probably rename this one `SplitPanelLeft`

Reply:
Yeah agreed, done